### DISCUSSION TITLE
[oracle]想实现在“数据字典”中展示Oracle的内容

### DISCUSSION THREAD

**Discussion by unknowissue** [hhyo/Archery#1338] [Ideas]
我想实现在“数据字典”中展示Oracle的内容
目前遇到的问题是data_dictionary.html中table 表头定义的是MySQL的属性，这个我想成写活的，判断一下db_type，MySQL原样显示，Oracle显示其他表头
这个有啥方案么？

**Comment by hhyo** [comment]
看了一下，除了meta_data的信息，其他都是动态列名，如果整体展示有较大差异，可以不同的数据库采取不同的 Modal ，否则适配下meta_data就行

**Reply by unknowissue** [reply]
我弄了一下
https://github.com/hhyo/Archery/pull/1351
帮忙看看行不行，谢谢
功能是可以实现的。

**Comment by unknowissue** [comment]
另外，我增加了  pip3 install pandas 依赖，
这个做ci时，我该在哪里添加条目，使这个 pandas 依赖 能自动安装？

**Reply by LeoQuote** [reply]
我的天🤦‍♀️ 要pandas吗，会不会太大啊

**Reply by LeoQuote** [reply]
哦还好还好，requirements里加就行了

**Reply by unknowissue** [reply]
> 我的天🤦‍♀️ 要pandas吗，会不会太大啊

嗯，主要是要是和MySQL一样的循环的话，表和字段数量一多，性能堪忧，导出页面挂了，还没导出来。
